GVFS/GVFS.FunctionalTests/Tools/GVFSFunctionalTestEnlistment.cs

```diff
@@ -115,7 +115,18 @@ public static string GetUniqueEnlistmentRootWithSpaces()
         public string GetObjectRoot(FileSystemRunner fileSystem)
         {
             IEnumerable<FileSystemInfo> localCacheRootItems = this.LocalCacheRoot.ShouldBeADirectory(fileSystem).WithItems();
-            localCacheRootItems.Count().ShouldEqual(2, "Expected local cache root to contain 2 items. Actual items: " + string.Join(",", localCacheRootItems));
+
+            FileInfo[] files = localCacheRootItems.Where(info => info is FileInfo).Cast<FileInfo>().ToArray();
+            files.Where(f => string.Equals(f.Name, "mapping.dat", StringComparison.OrdinalIgnoreCase))
+                 .Count()
+                 .ShouldEqual(1, "Local cache root should contain a single 'mapping.dat' file, actual files: " + string.Join<FileInfo>(",", files));

```

Comment:
This count can only ever be 1 or 0 right? This looks like an indirect way to check `fileSystem.FileExists(Path.Combine(this.LocalCacheRoot, "mapping.dat")).ShouldBeTrue()`. Or we could stick to the other pattern with `this.LocalCacheRoot.ShouldBeADirectory(fileSystem).WithFile("mapping.dat")`. I think either of those would make the intent of the test more clear.

Reply:
Good point, I got caught up in how the check was working before.  It's much cleaner to use the helpers we already have.